src/main/java/uk/gov/justice/digital/delius/transformers/OffenderTransformer.java

```diff
@@ -4,21 +4,43 @@
 import com.google.common.collect.ImmutableMap;
 import org.springframework.beans.factory.annotation.Autowired;
```

Comment:
IntelliJ suggested a ton of changes in this file that I accepted, sorry, totally unrelated to my change which is at the end